providers/jira/JIRAConstants.groovy

```diff
@@ -27,9 +28,10 @@ public class JIRAConstants {
 	static def PROVIDER_DESCRIPTION =
 	[
 		'JIRA provider. Configuration fields are below',
-		'(Required)' + SERVER_URL    + ': the JIRA URL to connect to',
-		'(Required)' + USERNAME      + ': User name',
-		'(Required)' + PASSWORD      + ': Password',
+		'(Required)' + SERVER_URL    + ': The JIRA URL to connect to',
+		'(Optional)' + TOKEN         + ': A personal access token. Required if not using username and password.',

```

Comment:
Do we need to change (Optional) to (Recommended) to be consistent with README.md?

Reply:
Yeah, good point. Change made.